### PR TITLE
Added Leanplum support

### PR DESCRIPTION
**Pull request by BenchR267**

### CODE REVIEW

File: Providers/LeanplumProvider.h [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#import "ARAnalyticalProvider.h"
+
+@interface LeanplumProvider : ARAnalyticalProvider
+
+- (id)initWithIdentifier:(NSString *)identifier NS_UNAVAILABLE;
+
+/**
+ * Initializes an instance of LeanplumProvider with the given parameters.
+ * 
+ * NOTE: developmentKey XOR productionKey has to be set!
```
Comment (orta) [inline Providers/LeanplumProvider.h]: 👍

**Comment by orta** [comment]
Yep, looks good to go on green, 👍

**Comment by BenchR267** [comment]
awesome, thanks! :)

**Comment by orta** [comment]
Shipped to trunk too 👍

**Comment by BenchR267** [comment]
Super! :)

**Pull request merged**